### PR TITLE
Clarify subtitle

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: src/components/settings/ApiSettingsBase.tsx [hunk 1]
```diff
@@ -76,6 +77,7 @@ class ApiSettingsBase extends React.Component<Props, State> {
     let cardActions = (
       <CardActions>
         <Button variant="contained" onClick={() => this.generateNewAccessToken()}>
```
Comment (fkorotkov) [inline src/components/settings/ApiSettingsBase.tsx]: Should it be?

```suggestion
        <Button variant="contained" onClick={() => this.generateNewAccessToken()} startIcon={<Https />}>
```

File: src/components/settings/ApiSettingsBase.tsx [hunk 2]
```diff
@@ -75,7 +76,8 @@ class ApiSettingsBase extends React.Component<Props, State> {
     }
     let cardActions = (
       <CardActions>
-        <Button variant="contained" onClick={() => this.generateNewAccessToken()}>
+        <Button variant="contained" onClick={() => this.generateNewAccessToken()} startIcon={<Https />}>
+          <Https />
```
Comment (RDIL) [inline src/components/settings/ApiSettingsBase.tsx]:
```suggestion
```

**Comment by fkorotkov** [comment]
Typecheck is failing :-(

**Comment by RDIL** [comment]
Yeah I see

**Pull request merged**